virttest/qemu_devices/qcontainer.py

```diff
@@ -1718,6 +1719,145 @@ def images_define_by_params(self, name, image_params, media=None,
                                                    "bus_extra_params"),
                                                image_params.get("force_drive_format"))
 
+    def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
+                                    bus_type=None, serial_name=None,
+                                    bus=None, nr=None, reg=None):
```

Comment:
Test pass now.
 Only one concern, why do not keep consistent with other serial option, i mean, `serial_name`, but not `serial_reg`, `serial_nr` ?